go/pkg/client/tree.go

```diff
@@ -112,6 +113,18 @@ func getRelPath(base, path string) (string, error) {
 	return rel, nil
 }
 
+// overrideWorkingDir changes workingDir component of execRoot relative path
```

Comment:
> // overrideWorkingDir changes workingDir component of execRoot relative path

workingDir component of execRoot relative path?

I think this statement is a bit confusing - workingDir shouldn't be part of the exec root.

Reply:
Updated the comment and renamed the function to make it role more straightforward